internal/process/process.go

```diff
@@ -0,0 +1,51 @@
+// Package process scans a dir for media files, and some pre-set conditions
+// to determine which media files should have a job created for them to later
+// be evaluated by Twilight
+package process
+
+import (
+	"os"
+	"path/filepath"
+)
+
+// TODO(jaredallard): running files through ffprobe is likely to end up
+// working better than file extensions
+// valid media file extensions
+var mediaExts = map[string]bool{
+	".mp4":  true,
+	".mkv":  true,
+	".mov":  true,
+	".webm": true,
+}
+
+// ProcessDir finds media files in a dir and returns their paths
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [comment on exported function Dir should be of the form "Dir ..."](https://app.codacy.com/gh/tritonmedia/downloader-go/pullRequest?prid=5012618)